<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginUI</class>
 <widget class="QDialog" name="LoginUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54)</string>
  </property>
  <widget class="QLabel" name="login">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_email">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>204</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt; color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_password">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt; color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_textbox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_textbox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>285</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt;
color: rgb(255, 255, 255);
background-color: rgb(189, 189, 189);
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="noacc">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>340</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 241, 241)</string>
   </property>
   <property name="text">
    <string>Don't have an account?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createacc_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>337</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 241, 241)</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
